lib/sql.js

```diff
@@ -2,19 +2,29 @@ var util = require('util');
 var async = require('async');
 var assert = require('assert');
 var Connector = require('./connector');
+var debug = require('debug')('loopback:connector:sql');
+var ParameterizedSQL = require('./parameterized-sql');
 
 module.exports = SqlConnector;
 
 /**
- * Base class for connectors that are backed by relational databases/SQL
+ * Base class for connectors that connect to relational databases using SQL
  * @class
  */
 function SqlConnector() {
+  // Call the super constructor
   Connector.apply(this, [].slice.call(arguments));
 }
 
+// Inherit from the base Connector
 util.inherits(SqlConnector, Connector);
 
+// Export ParameterizedSQL
+SqlConnector.ParameterizedSQL = ParameterizedSQL;
```

Comment:
For the sake of consistency, please consider using `ParameterizedSql`. The connector class name uses `Sql` instead of `SQL` too.